vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1675,10 +1675,12 @@ class RoomDetailFragment @Inject constructor(
             shareText(requireContext(), action.messageContent.body)
         } else if (action.messageContent is MessageWithAttachmentContent) {
             lifecycleScope.launch {
```

Comment:
why not using viewLifecycleScope? isAdded check would not be needed then

Reply:
yeah, we will change that later, I'm not even sure if this isAdded statement is still necessary with `lifecycleScope`